src/ogc/wcs/WcsCapabilities.js

```diff
@@ -61,6 +61,20 @@ define([
             this.assembleDocument();
         };
 
+        /**
+         * Returns the GetCoverage base url as detailed in the capabilities document
+         * @param coverageId the coverage id or name
+         */
+        WcsCapabilities.prototype.getCoverageBaseUrl = function () {
+            if (this.version === "1.0.0") {
+                return this.capability.request.getCoverage.get
```

Comment:
Is there a typo on this line?

Reply:
All I'm seeing is the missing semicolon, was there a different error?

Reply:
I can't make sense of `getCoverage.get`

Reply:
The HTTP "GET" url as opposed to the "POST". Would it be better to change the names for 1.0.0 to look like `.methodGet` and `.methodPost`?

Reply:
I see. Indeed the object names look like function names. Just do whatever you think is most consistent with WMS, and we can sort out any nuances like this later.